bin/GameIndex.dbf

```diff
@@ -1735,10 +1734,10 @@ Region = PAL-M8
 Compat = 5
 XgKickHack = 1
 [patches = 80802EA9]
-comment= Patch By Prafull
-// fix hang at loading screen
-patch=1,EE,002aa934,word,00000000
-patch=1,EE,002aa574,word,00000000
+	comment= Patch By Prafull
```

Comment:
You should fix the `comment=` lines so that they're in the same style, like in this case removing the space between = and Patch and changing `By` to the lower-case `by`

`comment=Patch by Prafull`